src/solvers/flattening/bv_pointers.cpp

```diff
@@ -68,58 +68,33 @@ void bv_endianness_mapt::build_big_endian(const typet &src)
 endianness_mapt
 bv_pointerst::endianness_map(const typet &type, bool little_endian) const
 {
-  return bv_endianness_mapt{type, little_endian, ns, bv_pointers_width};
+  return bv_endianness_mapt{type, little_endian, ns, bv_width};
```

Comment:
That second commit is a part-revert of 5cf2b07c12f. If we go for removing the capability that the commit was trying to add we should ponder a clean revert instead.